### PR TITLE
Retire dashboards management

### PR DESCRIPTION
**Pull request by raintygao**
### Description

Retire dashboards management

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/advanced_settings/public/plugin.ts [hunk 1]
```diff
@@ -61,21 +60,6 @@ export class AdvancedSettingsPlugin
       },
     });
 
-    if (home) {
```
Comment (wanglam) [inline src/plugins/advanced_settings/public/plugin.ts:64]: Do we need to remove the definition of the home variable?

File: src/plugins/saved_objects_management/public/plugin.ts [hunk 2]
```diff
@@ -177,41 +175,6 @@ export class SavedObjectsManagementPlugin
     const columnSetup = this.columnService.setup();
     const namespaceSetup = this.namespaceService.setup();
 
-    if (home) {
```
Comment (wanglam) [inline src/plugins/saved_objects_management/public/plugin.ts:180]: Prefer clear unused variables.

File: src/plugins/advanced_settings/public/plugin.ts [hunk 3]
```diff
@@ -61,21 +60,6 @@ export class AdvancedSettingsPlugin
...
-          defaultMessage:
-            'Customize your OpenSearch Dashboards experience — change the date format, turn on dark mode, and more.',
-        }),
-        icon: 'gear',
-        path: '/app/management/opensearch-dashboards/settings',
-        showOnHomePage: false,
-        category: FeatureCatalogueCategory.ADMIN,
-      });
-    }
-
```
Comment (Hailong-am) [inline src/plugins/advanced_settings/public/plugin.ts:78]: i think we should keep this and update path to `app/settings`

File: src/plugins/saved_objects_management/public/plugin.ts [hunk 4]
```diff
@@ -177,41 +175,6 @@ export class SavedObjectsManagementPlugin
...
-        description: i18n.translate('savedObjectsManagement.objects.savedObjectsDescription', {
-          defaultMessage:
-            'Import, export, and manage your saved searches, visualizations, and dashboards.',
-        }),
-        icon: 'savedObjectsApp',
-        path: '/app/management/opensearch-dashboards/objects',
-        showOnHomePage: false,
-        category: FeatureCatalogueCategory.ADMIN,
-      });
-    }
```
Comment (Hailong-am) [inline src/plugins/saved_objects_management/public/plugin.ts:195]: same here. i think we should not remove this

Comment (raintygao) [reply src/plugins/saved_objects_management/public/plugin.ts:195]: Updated. Do you mean we should update path like this?

Comment (Hailong-am) [reply src/plugins/saved_objects_management/public/plugin.ts:195]: should this be `app/objects` ?

File: src/plugins/saved_objects_management/opensearch_dashboards.json [hunk 5]
```diff
@@ -13,5 +13,5 @@
     "visAugmenter"
   ],
   "extraPublicDirs": ["public/lib"],
-  "requiredBundles": ["opensearchDashboardsReact", "home"]
+  "requiredBundles": ["opensearchDashboardsReact","home"]
```
Comment (Hailong-am) [inline src/plugins/saved_objects_management/opensearch_dashboards.json:16]: nit: missing space here?

File: src/plugins/advanced_settings/opensearch_dashboards.json [hunk 6]
```diff
@@ -5,5 +5,5 @@
   "ui": true,
   "requiredPlugins": ["management"],
   "optionalPlugins": ["home"],
-  "requiredBundles": ["opensearchDashboardsReact", "home"]
+  "requiredBundles": ["opensearchDashboardsReact","home"]
```
Comment (Hailong-am) [inline src/plugins/advanced_settings/opensearch_dashboards.json:8]: nit: missing space here?

File: src/plugins/saved_objects_management/public/plugin.ts (hunk 4 above)
Comment (Hailong-am) [reply src/plugins/saved_objects_management/public/plugin.ts:195]: @raintygao sorry for misunderstanding here. it is `/app/objects`.

Comment (raintygao) [reply src/plugins/saved_objects_management/public/plugin.ts:195]: Never mind. It's my problem.

**Review by wanglam — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]

**Pull request merged**